WPMedia/fusion-news-theme-blocks
# PEN-1763 - Regression - Update header links bar

## Description

Fix issue where when links wrap line-height changes
Move separator to be before all but the first item to fix the wrapping issue making it show the separator after the link item

## Jira Ticket
- [PEN-1763](https://arcpublishing.atlassian.net/browse/PEN-1763)

## Test Steps

1. Checkout this branch `git checkout PEN-1763-styling-regression`
2. In Fusion repo run with header nav chain block linked `npx fusion start -f -l @wpmedia/header-nav-chain-block`
3. Navigate to the article page - http://localhost/homepage/?_website=the-gazette
4. Verify there is spacing between the link items in the navigation chain
5. Verify when the screen size is smaller than the needed size to fit all on the navigation chain the links disappear including the separator


## Effect Of Changes
### Before

Links that wrapped move the line placement and leave the separator showing after link item

<img width="1111" alt="PEN-1764-before" src="https://user-images.githubusercontent.com/868127/107974008-dea03a00-6fad-11eb-837e-ff4f1bb9701a.png">


### After

When a link wraps the line placement stays and no separator after the last link item

<img width="1066" alt="PEN-1764-after" src="https://user-images.githubusercontent.com/868127/107974071-f4156400-6fad-11eb-914c-3e5ca70749e7.png">


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
